### PR TITLE
WF-65091-Address formatting issues in DocIO and Presentation

### PR DESCRIPTION
**Pull request by sharonaumao**
<table style="width: 521.306px;">
<tbody>
<tr>
<td style="width: 156px;">Task Link</td>
<td style="width: 349.306px;">https://syncfusion.atlassian.net/browse/WF-65091</td>
</tr>
<tr>
<td style="width: 156px;">Description</td>
<td style="width: 349.306px;">Resolved formatting issues in Presentation UG</td>
</tr>
<tr>
<td style="width: 156px;">Solution</td>
<td style="width: 349.306px;">Resolve formatting and hyperlink issues in mail-merge-options, Iterating-Word-document-elements, Accepting-or-Rejecting-Track-Changes, FAQ, Loading-and-Saving-document, word-to-pdf, Working-with-Charts, Working-With-Content-Controls, Working-with-Mathematical-Equation, Working-with-Paragraph, Working-with-Table-Of-Contents, Working-with-Word-document, create-read-edit-master-slide-and-layout-slide-in-c-sharp, Presentation-to-PDF</td>
</tr>
<tr>
<td style="width: 156px;">Content review confirmation</td>
<td style="width: 349.306px;">Yes</td>
</tr>
</tbody>
</table>

[WF-65091]: https://syncfusion.atlassian.net/browse/WF-65091?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md [hunk 1]
```diff
@@ -11,13 +11,12 @@ documentation: UG
...
-<li>Access the [MasterSlide](https://help.syncfusion.com/cr/file-formats/Syncfusion.Presentation.IMasterSlide.html) in PowerPoint file.</li>
-<li>Add [LayoutSlide](https://help.syncfusion.com/cr/file-formats/Syncfusion.Presentation.ILayoutSlide.html) to the [MasterSlide](https://help.syncfusion.com/cr/file-formats/Syncfusion.Presentation.IMasterSlide.html).</li>
-<li>Customize the [LayoutSlide](https://help.syncfusion.com/cr/file-formats/Syncfusion.Presentation.ILayoutSlide.html).</li>
-<li>Create a slide with 9 pre-defined layout slides.</li>
-<li>Customize the layout slides to fit your own scenarios.</li>
-</ol>
+
+1. Access the [MasterSlide](https://help.syncfusion.com/cr/file-formats/Syncfusion.Presentation.IMasterSlide.html) in PowerPoint file.
+2. Add [LayoutSlide](https://help.syncfusion.com/cr/file-formats/Syncfusion.Presentation.ILayoutSlide.html) to the [MasterSlide](https://help.syncfusion.com/cr/file-formats/Syncfusion.Presentation.IMasterSlide.html).
+3. li>Customize the [LayoutSlide](https://help.syncfusion.com/cr/file-formats/Syncfusion.Presentation.ILayoutSlide.html).
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md:17]: Remove list tag properly in this line

File: File-Formats/DocIO/word-to-pdf.md [hunk 2]
```diff
@@ -1406,7 +1406,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 This setting allows you to set the PDF conformance level.
 
-The following code sample shows how to set the PdfConformanceLevel while converting Word to PDF.
+The following code sample shows how to set the [PdfConformanceLevel](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.PdfConformanceLevel.html) while converting Word to PDF.
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/word-to-pdf.md:1409]: Use this link

https://help.syncfusion.com/cr/file-formats/Syncfusion.DocToPDFConverter.DocToPDFConverterSettings.html#Syncfusion_DocToPDFConverter_DocToPDFConverterSettings_PdfConformanceLevel

File: File-Formats/DocIO/FAQ.md [hunk 3]
```diff
@@ -2704,7 +2704,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## How to enable track changes for Word document?
 
-TrackChanges is used to keep track of the changes made to a Word document. This can be enabled by using the TrackChanges property of the Word document.
+[TrackChanges](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.WordDocument.html#Syncfusion_DocIO_DLS_WordDocument_TrackChanges) is used to keep track of the changes made to a Word document. This can be enabled by using the [TrackChanges](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.WordDocument.html#Syncfusion_DocIO_DLS_WordDocument_TrackChanges) property of the Word document.
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/FAQ.md:2707]: Remove link from first TrackChanges word

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md (hunk 1 above)
Comment (sharonaumao) [reply File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md]: @Suriya-Balamurugan -I have addressed the feedback

File: File-Formats/DocIO/word-to-pdf.md (hunk 2 above)
Comment (sharonaumao) [reply File-Formats/DocIO/word-to-pdf.md]: @Suriya-Balamurugan -It now added

File: File-Formats/DocIO/FAQ.md (hunk 3 above)
Comment (sharonaumao) [reply File-Formats/DocIO/FAQ.md]: @Suriya-Balamurugan -I have addressed the feedback

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**